index.js

```diff
@@ -0,0 +1,210 @@
+const inquirer = require("inquirer");
+const fs = require("fs");
+
+// All Licenses Obj, with categories. Source: https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba (better be imported via an API)
+
+const LICENSES = {
```

Comment:
if you want ot push yourself, consider placing this in a separate file, perhaps in a constants folder and importing it into this file to improve readability.

Reply:
Always use camelcasing for js vars